Assets/Scripts/Utilities/CCLogger.cs

```diff
@@ -270,7 +270,7 @@ private async Task _logAsync(string eventType, string msg)
         string json = JsonUtility.ToJson(this, true);
 
         #if UNITY_EDITOR
```

Comment:
Let's remove 272 - 274

Reply:
It's hard to see here, but this request came because the logging on 270 was commented out, and is probably no longer needed.

Reply:
Gotcha, thanks, will do!